### ISSUE TITLE
Custom driver path installation does not work for chrome in 2.0.0

### ISSUE THREAD

**Issue by nberezina** [SergeyPirogov/webdriver_manager#82]
Today I got webdriver-manager upgraded to 2.0.0 from 1.8
I am using statement
```
driverLoc = "C:\Webdriver"
DriverPath = ChromeDriverManager().install(driverLoc)
```
Driver path returned  "C:\Webdriver"but actual webdriver installed to c:\users\my_username\.wdm\...
Rolling back to 1.8 for now...

**Comment by SergeyPirogov** [comment]
@nberezina it is a bug 🐜

**Comment by SergeyPirogov** [comment]
@nberezina Fixed in 2.1.0. Now you can put custom path as 

```
ChromeDriverManager(path="C:\Webdriver").install()
```
